component/src/main/java/org/wso2/extension/siddhi/execution/string/EqualsIgnoreCaseFunctionExtension.java

```diff
@@ -42,7 +42,7 @@
 @Extension(
         name = "equalsIgnoreCase",
         namespace = "str",
-        description = "Compares two strings lexicographically.",
+        description = "Returns a boolean value by comparing two strings lexicographically without considering the letter case.",
```

Comment:
We need to check whether this line length issue cause for the build failing issue.
We can verify that by building the projects once we have done the changes.